frontend/public/locales/en/letters.json

```
@@ -6,7 +6,7 @@
       "letters": "Letters"
     },
     "subject": "Subject",
-    "date-sent": "Date sent",
+    "date": "Date:",
```

Use translation function interpolation.

```suggestion
    "date": "Date: {{date}}",
```